composer/algorithms/cutmix/cutmix.py

```diff
@@ -61,6 +62,9 @@ def cutmix_batch(X: Tensor,
         bbox: optional, predetermined (rx1, ry1, rx2, ry2) coords of the bounding box.
         indices: Permutation of the batch indices `1..B`. Used
             for permuting without randomness.
+        uniform_sampling: If true, sample the bounding box such that each pixel
```

Comment:
```suggestion
        uniform_sampling (bool, optional): If true, sample the bounding box such that each pixel
```

want to clean up these docstrings while you are at it? 😄